### PR TITLE
core: pagerduty info for projects

### PR DESCRIPTION
**Pull request by mikecutalo**
### Description
Adds Pagerduty information for a given project if applicable, the concepts used track Pagerduty definitions. In this case `service_ids` tracks 1:1 with what Pagerduty calls them.s

### CODE REVIEW

**Review by scarlettperry — APPROVED** [review]

File: api/core/project/v1/project.proto [hunk 1]
```diff
@@ -29,3 +30,10 @@ message ProjectDependencies {
 message Dependency {
   repeated string ids = 1;
 }
+
+// Pagerduty specific information for a project
+message Pagerduty {
```
Comment (danielhochman) [inline api/core/project/v1/project.proto:35]: super nit: PagerDuty

File: api/core/project/v1/project.proto [hunk 2]
```diff
@@ -17,6 +17,7 @@ message Project {
   // Allows an organization to populate any information they see fit that does not fit our schema
   map<string, google.protobuf.Value> data = 5;
   ProjectDependencies dependencies = 6;
+  Pagerduty pagerduty = 7;
```
Comment (danielhochman) [inline api/core/project/v1/project.proto:20]: should we nest this in an oncall block or something? just in case we expand for future non-pagerduty

**Review by danielhochman — APPROVED** [review]
lgtm, two minor comments

File: api/core/project/v1/project.proto (hunk 2 above)
Comment (mikecutalo) [reply api/core/project/v1/project.proto:20]: Or we could do an `Alerts` message with any type of information which could also include oncall? Or maybe the word `Alerts` is too specific.

File: api/core/project/v1/project.proto [hunk 3]
```diff
@@ -17,6 +17,7 @@ message Project {
   // Allows an organization to populate any information they see fit that does not fit our schema
   map<string, google.protobuf.Value> data = 5;
   ProjectDependencies dependencies = 6;
+  Alerts alerts = 7;
```
Comment (danielhochman) [inline api/core/project/v1/project.proto:20]: Alerts sounds like it's going to list out the Alerts

**Review by danielhochman — APPROVED** [review]

**Pull request merged**